docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -411,14 +406,39 @@
             <row>
               <entry><emphasis role="bold">COMPRESSED</emphasis></entry>
 
-              <entry>Specifies the type of compression used. If omitted, the
-              default is LZW, a variant of the Lempel-Ziv-Welch algorithm.
-              Specifying ROW compresses index entries based on differences
-              between contiguous rows (for use with fixed-length records,
-              only), and is recommended for use in circumstances where
-              speedier decompression time is more important than the amount of
-              compression achieved. FIRST compresses common leading elements
-              of the key (recommended only for timing comparison use).</entry>
+              <entry>Optional. Specifies the index should be compressed using
+              the type of compression specified.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>option</emphasis></entry>
+
+              <entry>If omitted, the default is <emphasis
```

Comment:
Better to start with an introductory statement similar to before.  "This option specifies the type of compression used"
I would consider restructuring this i) so it is more clearly of a set of options ii) so the common cases are described first.

E.g.
Specifies the type of compression used. If omitted, the default is LZW.  The following options are supported:

* LZW....
* 'inplace'.  causes the index to be built using the new inplace compression format.  Advantages? Disadvantages?  Explicitly state that it is supported on versions 9.0 and later.
* 'inplace:lz4hc'.  How this differs from above.
* ROW
* FIRST??  Although techinically legal no one would ever use this.  I would remove.

Note technically the syntax of the string is:

"<compression>[:option,option,option]".  One of the options for the inplace compression is the compression used for the payload.  "inplace" is equivalent to "inplace:lz4" or "inplace:compression=lz4".  Other values that can be used are inplace:lzw (not recommended) and inplace:row for fixed size indexes or inplace:uncompressed for no compression.

To cover this fully you may need to split it out into another section.  E.g. there is an option to control the minimum compression percentage that must be obtained - and to use uncompressed if that is not achieved.  

'inplace:compressThresholdPercent=95'

It depends if you want to document all the options/syntax or just the most useful.  I think in this case indicating that there is support for options is useful, but detailing them may not be (since generally they will not be used.)

Reply:
Gavin, you said " Advantages? Disadvantages?" -- can you please specify what those are so Jim can include that information? 
And, does inplace compress only search terms, or search terms and payload? -- Same question for LZW and ROW, too.

Reply:
inplace uses two different compression algorithms.  One is always used for the keyed portion.  The compression of the payload is configured with those extra options.
LZW and ROW compress the keyed and payload together.